azure_devops_rust_api/examples/get_commits.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// get_commits.rs
```

Comment:
naming: I've been naming the current examples with the module/feature as a prefix (in this case "git") to help group them by feature.  Please could you rename this to `git_pr_commits.rs`.

Reply:
Updated in the latest [commits](https://github.com/microsoft/azure-devops-rust-api/pull/27/commits/276e698c731ca453693e3800bab1ecbdad2338dd)